gecko/compose/workflow/engine.py

```
@@ -306,6 +309,38 @@ async def execute(
                 raise WorkflowError(f"Workflow execution failed: {e}") from e
             raise
 
+    def _cleanup_history(self, context: WorkflowContext, max_steps: int = 20):
+        """
+        [P1-2 修复] 定期清理 history，防止无界增长
```

Ruff (0.14.7)

Docstring contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF002)